app/components/channel_modal.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component, PropTypes} from 'react';
+import {Animated, Dimensions} from 'react-native';
+
+const AnimatedView = Animated.View;
```

Comment:
@jarredwitt A tiny nitpick: If we're using aliased destructuring to get the window dimensions on the next line, wouldn't it be more consistent here to do `const {View: AnimatedView} = Animated;`?